src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -116,20 +113,28 @@ def run_offline_reintegration_test(self, num_pool, data=False):
             self.pool.display_pool_daos_space("Pool space: Beginning")
             pver_begin = self.get_pool_version()
             self.log.info("Pool Version at the beginning %s", pver_begin)
-            output = self.dmg_command.pool_exclude(self.pool.uuid,
-                                                   rank)
+            if server_boot is False:
+                output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                       rank)
+            else:
+                output = self.dmg_command.system_stop(ranks=rank)
+                self.pool.wait_for_rebuild(True)
+                self.log.info(output)
+                output = self.dmg_command.system_start(ranks=rank)
+
             self.log.info(output)
             self.is_rebuild_done(3)
             self.assert_on_rebuild_failure()
```

Comment:
Minor: If you want this can be replaced by 
`self.pool.wait_for_rebuild(True)` as well.